### DISCUSSION TITLE
SmartAutoClicker-1.2.2

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#65] [Announcements]
# New features:
* Event copy UI is now similar to the action copy one.
* It is now possible to copy an Event from another scenario.
* Similar events/actions are now merged in copy screen.
* Search bar has been added to the copy screens.

# Bug fixes:
* Fixes some weird behaviours on edit text.
* Fixes the invalid state of the OK button in the event and action configuration dialogs.
* Fixes a crash on some devices that can occurs when capturing a new condition several times.
* Fixes a crash on slow devices when creating/moving an action.

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/1.2.2'>SmartAutoClicker-1.2.2</a>.</em>

**Comment by ladead** [comment]
Thank you so much this update has fixed all the crashing and broken auto clicking referenced in issues a couple times and fixed so many tiny little things like names defaulting and more then one actions breaking

**Comment by Nain57** [comment]
Version 1.2.2 is now available on F-Droid: https://f-droid.org/fr/packages/com.buzbuz.smartautoclicker/
